deployer/scripts/util.sh

```diff
@@ -514,7 +517,7 @@ function wait_for_fluentd_to_catch_up() {
     else
         echo failed - $FUNCNAME: not found $expected record project logging for $uuid_es after $timeout seconds
         echo "Checking journal for $uuid_es..."
-        if journalctl | grep $uuid_es ; then
+        if sudo journalctl | grep $uuid_es ; then
```

Comment:
I was getting some errors that not all log entries could be read.  Is it a problem to use sudo here?